### PR TITLE
Qt: fix stylesheets on various platforms

### LINKED ISSUE #9732: Regression: YoRHa theme background does not display (#9728)
#9728 broke the background for me on the YoRHa theme at least on AppImage (Manjaro Gnome 20.2.1)

---

## Before
Screenshot from 2021-02-06 20-18-06

## After 
image

### PR DESCRIPTION
**Pull request by Nekotekina**
Replace relative pathes to resources.
Should fix #9732

### CODE REVIEW

**Comment by AniLeo** [comment]
Please leave this open for a bit, I can't reproduce the original issue for some reason, need to figure out how to reproduce it again

**Comment by AniLeo** [comment]
Ok, managed to reproduce (I had placed a copy of the background in GuiConfigs in this PC manually), this PR fixes it indeed

File: rpcs3/rpcs3qt/gui_settings.cpp [hunk 1]
```diff
@@ -376,7 +378,17 @@ QString gui_settings::GetCurrentStylesheetPath()
 		QString path = dir.absoluteFilePath(stylesheet + ".qss");
 		QFile test(path);
 		if (test.exists())
-			return path;
+		{
+			test.open(QIODevice::ReadOnly);
+			std::string result = fs::get_cache_dir() + "temp.qss";
+			std::string sheet = test.readAll().toStdString();
+
+			// Fixup patches (replace resources with absolute patches) and store in temp file.
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/gui_settings.cpp:386]: Confusing patches and paths again..

**Comment by Megamouse** [comment]
Why do we need this complicated thing exactly?
And why is it used in the windows path?

**Comment by Nekotekina** [comment]
Using relative paths in qss doesn't guarantee correct work even on Windows, sadly.

**Comment by Nekotekina** [comment]
The only "reliable" alternative seems to be embedding those directly into the executable and use Qt fs prefix for resources. But it's no less complicates to configure.

**Comment by Megamouse** [comment]
Stylesheets aren't resources. They are just regular files anyone can add, change and play with.
There's no difference to any other file

File: rpcs3/rpcs3qt/gui_settings.cpp [hunk 2]
```diff
@@ -376,7 +378,17 @@ QString gui_settings::GetCurrentStylesheetPath()
 		QString path = dir.absoluteFilePath(stylesheet + ".qss");
 		QFile test(path);
 		if (test.exists())
-			return path;
+		{
+			test.open(QIODevice::ReadOnly);
+			std::string result = fs::get_cache_dir() + "temp.qss";
+			std::string sheet = test.readAll().toStdString();
+
+			// Fixup pathes (replace resources in GuiConfigs with absolute pathes) and store in temp file.
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/gui_settings.cpp:386]:
```suggestion
			// Fixup paths (replace resources in GuiConfigs with absolute paths) and store in temp file.
```

Comment (Nekotekina) [reply rpcs3/rpcs3qt/gui_settings.cpp]: Fixed.

**Pull request merged**